source/components/the-component-life-cycle.md

```diff
@@ -0,0 +1,48 @@
+When a component is invoked, a number of life-cycle hooks are triggered, in sequence, to properly initialize and render the component object and its corresponding DOM element.  The following hooks are a few of the most useful, for current apps.
+
+## init()
```

Comment:
good point!  Lets keep the guides dry as possible.

But do you think those little extra details are valuable though?  specifically that elementID is generated? and that arrays/objects are shared across instances?